runs.py

```diff
@@ -70,6 +70,10 @@ def won(self) -> bool:
     def killed_by(self) -> str | None:
         return self.data.get("killed_by")
 
+    @property
+    def ascension_level(self) -> int:
+        return self.data.get("ascension_level")
```

Comment:
I'm not sure if `"ascension_level"` can ever not be present, but if that's the case, it should be `.get(..., 0)` to make sure it's always an `int`.

Reply:
I've checked on my current run history of 307 runs and this key was always there. even on ascension 0, so I'm pretty sure, it is safe.

Reply:
Awesome! In which case this should be `self.data["ascension_level"]` (or just `self["ascension_level"]`); I'd rather an error raise in these cases than silently work.